game/definitions.rpy

```diff
@@ -972,6 +972,466 @@ init -990 python in jn_globals:
         "C.C"
     ]
 
+    # Natsuki hates these; awarding them as player names/nicknames detracts affinity/trust. Please forgive me.
+    # Source courtest of: https://github.com/RobertJGabriel/Google-profanity-words, with some additions by us
+    PROFANITY_LIST = {
+        "4r5e",
+        "5h1t",
+        "5hit",
+        "a_s_s",
+        "a55",
+        "aids",
+        "anal",
+        "anus",
+        "ar5e",
+        "arrse",
+        "arse",
+        "ass",
+        "asses",
+        "assfucker",
+        "ass-fucker",
+        "assfukka",
+        "asshole",
+        "assholes",
+        "asswhole",
+        "b!tch",
+        "b00bs",
+        "b17ch",
+        "b1tch",
+        "ballbag",
+        "balls",
+        "ballsack",
+        "bastard",
+        "beastial",
+        "beastiality",
+        "bellend",
+        "bestial",
+        "bestiality",
+        "bi+ch",
+        "biatch",
+        "bitch",
+        "bitcher",
+        "bitchers",
+        "bitches",
+        "bitchin",
+        "bitching",
+        "bloody",
+        "blow job",
+        "blowjob",
+        "blowjobs",
+        "boiolas",
+        "bollock",
+        "bollok",
+        "boner",
+        "boob",
+        "boobs",
+        "booobs",
+        "boooobs",
+        "booooobs",
+        "booooooobs",
```

Comment:
I get the feeling we should use regex here. This seems arbitrary and very easy to circumvent.